README.md

```diff
@@ -56,6 +56,9 @@ The operator can be deployed with Helm v3 chart in /charts as follows:
   * Disable persistence for basic testing. This can be done by adding `--set
     etcd.persistence.enabled=false` to the `helm install` command below.
 
+- If needed, configure secure communication between LINSTOR components.
+  [Read more below](#configuring-secure-communication-between-LINSTOR-components).
```

Comment:
This README is getting too long. Could you create `doc/security.md` or similar and move the encryption docs there?

Reply:
Replaced this part with a general step to check the security guide